doc/whats-new.rst

```diff
@@ -33,6 +33,10 @@ New Features
   of :py:class:`DataArray` and :py:class:`Dataset` objects and
   document the new method in :doc:`internals`. (:pull:`4248`).
   By `Justus Magin <https://github.com/keewis>`_.
+- Decode the CF attributes 'grid_mapping' and 'bounds' on file read
```

Comment:
~should update this list.~ Let's add a small para with the whole list to https://xarray.pydata.org/en/stable/weather-climate.html and link to that from here.

Also please take credit by adding your name and github ID if you would like.

Reply:
I added a paragraph, and I think I added a link.